Shared/Characters/PlayerBase.gd

```diff
@@ -13,6 +13,7 @@ var aim_mode: bool = false
 
 # Blocks any movement applied to it (includes rotation-movement)
 var block_movement: bool = true
+var block_input: bool = false
```

Comment:
if we have this now, do we need block_movement anymore?

Reply:
block_input blocks everything but movement, e.g. shooting or dashing. Needed for first round of second tutorial

Reply:
oh okay, then it's fine.